src/probability/integration.lean

```diff
@@ -131,7 +131,7 @@ begin
 end
 
 /-- The product of two independent, integrable, real_valued random variables is integrable. -/
-lemma indep_fun.integrable_mul {β : Type*} {mβ : measurable_space β} {X Y : α → β}
+lemma indep_fun.integrable_mul {β : Type*} [measurable_space β] {X Y : α → β}
```

Comment:
Why the change? Is it not usable with the implicit argument? I thought the `indep_fun` argument would provide the meausrable space.

Reply:
Yes, it doesn't work with the implicit argument (making it implicit breaks a proof in `variance.lean`) because it can't synthetize the `borel_space β` (maybe because it hasn't seen `indep_fun X Y μ` yet, I don't know). But in any case here there is no point in making it implicit, since beta will be the target space, like reals or ennreals, which already have a canonical measurable space structure (and since we require `borel_space β` below it's not like we really have a choice).